app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -757,6 +763,10 @@ export default class ComposerEditor extends Component {
   }
 
   _registerImageAltTextButtonClick(preview) {
+    if (!preview) {
+      return;
+    }
```

Comment:
why is this early return needed now?

Reply:
Not because of the composer v2 yet, but with this PR when we're displaying the form template, the preview element isn't even in the DOM anymore.

It was within `DEditor` before, but we don't render it anymore for the form template.

This is signal of more needed decoupling for this but one step at a time 😄